makefiles/network-cmds.mk

```diff
@@ -5,91 +5,47 @@ endif
+		https://raw.githubusercontent.com/apple-oss-distributions/xnu/xnu-8019.41.5/bsd/netinet/{mptcp_var,in_stat,in,tcp,tcp_var,ip_var,udp_var,if_ether,tcpip,icmp_var,igmp_var,tcp_seq,tcp_fsm,in_var,in_pcb}.h
 	@wget -q -nc -P $(BUILD_WORK)/network-cmds/include/netinet6 \
-		https://opensource.apple.com/source/xnu/xnu-6153.11.26/bsd/netinet6/ip6_var.h \
-		https://opensource.apple.com/source/xnu/xnu-6153.11.26/bsd/netinet6/in6_var.h \
-		https://opensource.apple.com/source/xnu/xnu-6153.11.26/bsd/netinet6/in6.h \
-		https://opensource.apple.com/source/xnu/xnu-6153.11.26/bsd/netinet6/nd6.h \
-		https://opensource.apple.com/source/xnu/xnu-6153.11.26/bsd/netinet6/mld6_var.h \
-		https://opensource.apple.com/source/xnu/xnu-6153.11.26/bsd/netinet6/in6_pcb.h \
-		https://opensource.apple.com/source/xnu/xnu-6153.11.26/bsd/netinet6/raw_ip6.h
+		https://raw.githubusercontent.com/apple-oss-distributions/xnu/xnu-8019.41.5/bsd/netinet6/{ip6_var,in6_var,in6,nd6,mld6_var,in6_pcb,raw_ip6}.h
 	@wget -q -nc -P $(BUILD_WORK)/network-cmds/include/sys \
-		https://opensource.apple.com/source/xnu/xnu-6153.11.26/bsd/sys/socket.h \
-		https://opensource.apple.com/source/xnu/xnu-6153.11.26/bsd/sys/unpcb.h \
-		https://opensource.apple.com/source/xnu/xnu-6153.11.26/bsd/sys/kern_event.h \
-		https://opensource.apple.com/source/xnu/xnu-6153.11.26/bsd/sys/kern_control.h \
-		https://opensource.apple.com/source/xnu/xnu-6153.11.26/bsd/sys/socketvar.h \
-		https://opensource.apple.com/source/xnu/xnu-6153.11.26/bsd/sys/sys_domain.h \
-		https://opensource.apple.com/source/xnu/xnu-6153.11.26/bsd/sys/mbuf.h \
-		https://opensource.apple.com/source/xnu/xnu-6153.11.26/bsd/sys/sockio.h
+		https://raw.githubusercontent.com/apple-oss-distributions/xnu/xnu-8019.41.5/bsd/sys/{proc_info,socket,unpcb,kern_event,kern_control,socketvar,sys_domain,mbuf,sockio}.h
+	@wget -q -nc -P$(BUILD_WORK)/network-cmds/include/mach \
+		https://raw.githubusercontent.com/apple-oss-distributions/xnu/xnu-8019.41.5/osfmk/mach/coalition.h
 
 	sed -i 's/#if INET6/#ifdef INET6/g' $(BUILD_WORK)/network-cmds/include/sys/sockio.h
+	sed -i 's|struct kevent_qos_s kqext_kev|struct kevent_qos_s { \n\
+             uint64_t        ident;          /* identifier for this event */ \n\
+             int16_t         filter;         /* filter for event */ \n\
+             uint16_t        flags;          /* general flags */ \n\
+             uint32_t        qos;            /* quality of service when servicing event */ \n\
+             uint64_t        udata;          /* opaque user data identifier */ \n\
+             uint32_t        fflags;         /* filter-specific flags */ \n\
+             uint32_t        xflags;         /* extra filter-specific flags */ \n\
+             int64_t         data;           /* filter-specific data */ \n\
+             uint64_t        ext[4];         /* filter-specific extensions */ \n\
+     }|g' $(BUILD_WORK)/network-cmds/include/sys/proc_info.h
```

Comment:
What's going on here? Care to explain? Should we keep network-cmds on a lower version for older iOS, like file-cmds?

Reply:
actually its defined in sys/event.h so I just need to delete this declaration entirely

Reply:
and I really don't know about iOS 12 considering I have one and only one testing device on iPadOS 14.3

Reply:
anyways the last time network-cmds got touched besides some changes that affect multiple makefiles is when checkra1n_memo is still a thing